pymc3/stats.py

```diff
@@ -465,9 +465,7 @@ def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
 
     Parameters
     ----------
-    traces : list of PyMC3 traces
-    models : list of PyMC3 models
-        in the same order as traces.
+    model_dict : dictionary of PyMC3 traces indexed by corresponding model
```

Comment:
I thought one could also just specify the model name directly, e.g. `{'glm': trace0, 'exp': trace1}`?

Reply:
No, we are just replacing a list of models and a corresponding list of traces with a dictionary. The model itself is required.